alf/utils/dist_utils.py

```diff
@@ -70,6 +70,11 @@ class AffineTransform(get_invertible(td.AffineTransform)):
     compatible with ``DistributionSpec.build_distribution()``.
     """
 
+    def __init__(self, loc, scale, event_dim=0, cache_size=0):
```

Comment:
Nit: If we never use `event_dim` and `cache_size`, perhaps we can just remove them from the arg list. That way, we'll always use pytorch's defaults?